src/components/auth/RegisterStudent.vue

```diff
@@ -0,0 +1,665 @@
+<!-- eslint-disable vue/valid-v-for -->
```

Comment:
I'm not sure which `v-for` is causing this error, but in general, I think we should try not to ignore these errors and track down which one is causing the error.

Reply:
Yeah, you are right, the indexes were mixing up. I fixed it